internal/pkg/types/message.go

```diff
@@ -29,6 +29,8 @@ const (
 	InvalidMethodID = MethodID(0xFFFFFFFFFFFFFFFF)
 	// SendMethodID is the method ID for sending money to an actor.
 	SendMethodID = MethodID(0)
+	// ConstructorMethodID is the method ID used to initialize an actor's state.
+	ConstructorMethodID = MethodID(1)
```

Comment:
An actor's constructor is always at method id 1. See the [VM actor spec](https://filecoin-project.github.io/specs/#systems__filecoin_vm__actor).